newrelic_lambda_cli/layers.py

```diff
@@ -69,10 +95,10 @@ def _add_new_relic(input, config, nr_license_key):
         if not layer["Arn"].startswith(utils.get_arn_prefix(aws_region))
     ]
 
-    new_relic_layers = []
+    new_relic_layer = []
```

Comment:
Why this change?

Reply:
because there is only ever one layer in the list. I thought it was clearer.